doc-content/enterprise-only/openshift/template-deploy-ha-resources-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='template-deploy-ha-resources-{context}-proc']
+= Configuring resource usage for a high-availability deployment
+
+If you are deploying the high-availability template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml`), you can optionally configure resource usage to optimize performance for your requirements.
+
+If you are deploying the single authoring environment template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml`), skip this procedure.
+
+.Prerequisites
+
+* You started the configuration of the template, as described in {startlink}.
+
+.Procedure
+
+Set any of the following parameters of the template as necessary:
+
+** *{CENTRAL} Container Memory Limit* (`{CENTRAL_CAPITAL_UNDER}_MEMORY_LIMIT`): The amount of memory requested in the OpenShift environment for the {CENTRAL} container. The default value is `8Gi`.
+** *{CENTRAL} JVM Max Memory Ratio* (`{CENTRAL_CAPITAL_UNDER}_JAVA_MAX_MEM_RATIO`): The percentage of container memory that is used for the Java Virtual Machine for {CENTRAL}. The default is `80`, for a limit of 80%. Set this parameter to `0` to use no limit.
```

Comment:
I think that no limit is not good to mention, because if it set to 0, then no limit is applied, but then is used default Xmx value (-Xmx1303m). So if somebody set it to 0 even if they had configured memory limit, they will have small java heap and could easily face Out of memory issues (pod restarts). 
```suggestion
** *{CENTRAL} JVM Max Memory Ratio* (`{CENTRAL_CAPITAL_UNDER}_JAVA_MAX_MEM_RATIO`): The percentage of container memory that is used for the Java Virtual Machine for {CENTRAL}. The default is `80`, for a limit of 80%.
```